### ISSUE TITLE
NSInternalInconsistencyException on insertRows

### ISSUE THREAD

**Issue by neilalexander** [neilalexander/seaglass#73]
## Specifications

  - Seaglass Version: 0.0.426-d3a661c
  - macOS Version: 10.13.6

## Expected Behavior

There should never be an `NSInternalInconsistencyException`.

## Actual Behavior

This happens really sporadically - sometimes a `NSInternalInconsistencyException` gets fired from `insertRows:withAnimation:` and causes Seaglass to crash.

## Steps to Reproduce the Problem

Not clear at this time.

**Comment by neilalexander** [comment]
Stack trace:
```
2018-09-14 14:54:28.026880+0100 Seaglass[10402:85794975] *** Assertion failure in -[NSTableRowData insertRowsAtIndexes:withRowAnimation:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1561.60.100/TableView.subproj/NSTableRowData.m:7470
2018-09-14 14:54:28.030145+0100 Seaglass[10402:85794975] [General] An uncaught exception was raised
2018-09-14 14:54:28.030163+0100 Seaglass[10402:85794975] [General] insertRowsAtIndexes:withRowAnimation: can not happen while updating visible rows!
2018-09-14 14:54:28.030222+0100 Seaglass[10402:85794975] [General] (
	0   CoreFoundation                      0x00007fff389102db __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x00007fff5fab1c76 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff38916072 +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x00007fff3aa3f340 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 193
	4   AppKit                              0x00007fff35f22273 -[NSTableRowData insertRowsAtIndexes:withRowAnimation:] + 304
	5   AppKit                              0x00007fff35f21ffa -[NSTableView _doUpdatedWorkWithHandler:] + 106
	6   AppKit                              0x00007fff35f21e72 -[NSTableView insertRowsAtIndexes:withAnimation:] + 162
	7   Seaglass                            0x00000001000f53d3 _T08Seaglass15MatrixRoomCacheC6insertySo7MXEventC_Si2attF + 1171
	8   Seaglass                            0x00000001000b6381 _T08Seaglass14MatrixServicesC15subscribeToRoomySS6roomId_tFySo7MXEventC_05SwiftB3SDK19MXTimelineDirectionOSo11MXRoomStateCtcfU2_ + 16129
	9   Seaglass                            0x00000001000b6d95 _T08Seaglass14MatrixServicesC15subscribeToRoomySS6roomId_tFySo7MXEventC_05SwiftB3SDK19MXTimelineDirectionOSo11MXRoomStateCtcfU2_TA + 53
	10  SwiftMatrixSDK                      0x0000000100f1b3cc _T0So15MXEventTimelineC14SwiftMatrixSDKE14listenToEventsypSayAC0A4TypeOGSg_ySo0A0C_AC19MXTimelineDirectionOSo11MXRoomStateCtctFyAJSg_SC02__jK0VANSgtcfU_TA + 76
	11  SwiftMatrixSDK                      0x0000000100f1b20a _T0So7MXEventCSgSC21__MXTimelineDirectionVSo11MXRoomStateCSgIegxyx_AceHIeyByyy_TR + 74
	12  SwiftMatrixSDK                      0x0000000100e4bd22 -[MXEventListener notify:direction:andCustomObject:] + 162
	13  SwiftMatrixSDK                      0x0000000100e5006a -[MXEventTimeline notifyListeners:direction:] + 407
	14  SwiftMatrixSDK                      0x0000000100e4f0ea -[MXEventTimeline addEvent:direction:fromStore:isRoomInitialSync:] + 1022
	15  SwiftMatrixSDK                      0x0000000100e4d594 -[MXEventTimeline paginate:direction:onlyFromStore:complete:failure:] + 1125
	16  SwiftMatrixSDK                      0x0000000100f1ace2 _T0So15MXEventTimelineC14SwiftMatrixSDKE8paginateSo15MXHTTPOperationCSgSu_AC19MXTimelineDirectionO9directionSb13onlyFromStoreyAC10MXResponseOyytGc10completiontF + 386
	17  Seaglass                            0x0000000100056896 _T08Seaglass22MainViewRoomControllerC06scrollC9DidScrollySo14NSNotificationCF + 4438
	18  Seaglass                            0x0000000100056e0c _T08Seaglass22MainViewRoomControllerC06scrollC9DidScrollySo14NSNotificationCFTo + 60
	19  CoreFoundation                      0x00007fff3889fedc __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
	20  CoreFoundation                      0x00007fff3889fdaa _CFXRegistrationPost + 458
	21  CoreFoundation                      0x00007fff3889fae1 ___CFXNotificationPost_block_invoke + 225
	22  CoreFoundation                      0x00007fff3885d880 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1664
	23  CoreFoundation                      0x00007fff3885c9b7 _CFXNotificationPost + 599
	24  Foundation                          0x00007fff3a96c8c7 -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
	25  AppKit                              0x00007fff35efe3d1 -[NSClipView _immediateScrollToPoint:] + 614
	26  AppKit                              0x00007fff35efe0a7 -[NSClipView scrollToPoint:] + 232
	27  AppKit                              0x00007fff36819139 -[NSTableRowData _keepTopRowStable:andDoWorkUntilDone:] + 675
	28  AppKit                              0x00007fff3681941d -[NSTableRowData _automaticRowHeightsUpdateVisibleRowViews] + 161
	29  AppKit                              0x00007fff35f27971 -[NSTableRowData updateVisibleRowViews] + 261
	30  AppKit                              0x00007fff35f3c469 -[NSTableView _scrollViewDidChangeBounds:] + 38
	31  CoreFoundation                      0x00007fff3889fedc __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
	32  CoreFoundation                      0x00007fff3889fdaa _CFXRegistrationPost + 458
	33  CoreFoundation                      0x00007fff3889fae1 ___CFXNotificationPost_block_invoke + 225
	34  CoreFoundation                      0x00007fff3885d880 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1664
	35  CoreFoundation                      0x00007fff3885c9b7 _CFXNotificationPost + 599
	36  Foundation                          0x00007fff3a96c8c7 -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
	37  AppKit                              0x00007fff35efe8b9 -[NSView _postBoundsChangeNotification] + 103
	38  AppKit                              0x00007fff35efe7e8 -[NSView translateOriginToPoint:] + 261
	39  AppKit                              0x00007fff35efe383 -[NSClipView _immediateScrollToPoint:] + 536
	40  AppKit                              0x00007fff35efe0a7 -[NSClipView scrollToPoint:] + 232
	41  AppKit                              0x00007fff35f39b7f -[NSScrollView scrollClipView:toPoint:] + 75
	42  AppKit                              0x00007fff35ee4929 -[NSClipView _scrollTo:animateScroll:flashScrollerKnobs:] + 1273
	43  AppKit                              0x00007fff364e4750 -[_NSScrollingConcurrentMainThreadSynchronizer _scrollToCanonicalOrigin] + 935
	44  AppKit                              0x00007fff36080d74 -[_NSScrollingConcurrentMainThreadSynchronizer _synchronize:completionHandler:] + 174
	45  AppKit                              0x00007fff36080c94 __80-[_NSScrollingConcurrentMainThreadSynchronizer initWithSharedData:constantData:]_block_invoke + 145
	46  libdispatch.dylib                   0x0000000101ca2d8f _dispatch_client_callout + 8
	47  libdispatch.dylib                   0x0000000101cb73d2 _dispatch_continuation_pop + 940
	48  libdispatch.dylib                   0x0000000101ca548f _dispatch_source_invoke + 658
	49  libdispatch.dylib                   0x0000000101caf053 _dispatch_main_queue_callback_4CF + 1135
	50  CoreFoundation                      0x00007fff388c8c19 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	51  CoreFoundation                      0x00007fff3888adfa __CFRunLoopRun + 2586
	52  CoreFoundation                      0x00007fff3888a153 CFRunLoopRunSpecific + 483
	53  HIToolbox                           0x00007fff37b74d96 RunCurrentEventLoopInMode + 286
	54  HIToolbox                           0x00007fff37b74b06 ReceiveNextEventCommon + 613
	55  HIToolbox                           0x00007fff37b74884 _BlockUntilNextEventMatchingListInModeWithFilter + 64
	56  AppKit                              0x00007fff35e25a73 _DPSNextEvent + 2085
	57  AppKit                              0x00007fff365bbe34 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 3044
	58  AppKit                              0x00007fff35e1a885 -[NSApplication run] + 764
	59  AppKit                              0x00007fff35de9a72 NSApplicationMain + 804
	60  Seaglass                            0x0000000100076ded main + 13
	61  libdyld.dylib                       0x00007fff606cb015 start + 1
	62  ???                                 0x0000000000000003 0x0 + 3
)
2018-09-14 14:54:28.032304+0100 Seaglass[10402:85794975] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'insertRowsAtIndexes:withRowAnimation: can not happen while updating visible rows!'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff389102db __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x00007fff5fab1c76 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff38916072 +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x00007fff3aa3f340 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 193
	4   AppKit                              0x00007fff35f22273 -[NSTableRowData insertRowsAtIndexes:withRowAnimation:] + 304
	5   AppKit                              0x00007fff35f21ffa -[NSTableView _doUpdatedWorkWithHandler:] + 106
	6   AppKit                              0x00007fff35f21e72 -[NSTableView insertRowsAtIndexes:withAnimation:] + 162
	7   Seaglass                            0x00000001000f53d3 _T08Seaglass15MatrixRoomCacheC6insertySo7MXEventC_Si2attF + 1171
	8   Seaglass                            0x00000001000b6381 _T08Seaglass14MatrixServicesC15subscribeToRoomySS6roomId_tFySo7MXEventC_05SwiftB3SDK19MXTimelineDirectionOSo11MXRoomStateCtcfU2_ + 16129
	9   Seaglass                            0x00000001000b6d95 _T08Seaglass14MatrixServicesC15subscribeToRoomySS6roomId_tFySo7MXEventC_05SwiftB3SDK19MXTimelineDirectionOSo11MXRoomStateCtcfU2_TA + 53
	10  SwiftMatrixSDK                      0x0000000100f1b3cc _T0So15MXEventTimelineC14SwiftMatrixSDKE14listenToEventsypSayAC0A4TypeOGSg_ySo0A0C_AC19MXTimelineDirectionOSo11MXRoomStateCtctFyAJSg_SC02__jK0VANSgtcfU_TA + 76
	11  SwiftMatrixSDK                      0x0000000100f1b20a _T0So7MXEventCSgSC21__MXTimelineDirectionVSo11MXRoomStateCSgIegxyx_AceHIeyByyy_TR + 74
	12  SwiftMatrixSDK                      0x0000000100e4bd22 -[MXEventListener notify:direction:andCustomObject:] + 162
	13  SwiftMatrixSDK                      0x0000000100e5006a -[MXEventTimeline notifyListeners:direction:] + 407
	14  SwiftMatrixSDK                      0x0000000100e4f0ea -[MXEventTimeline addEvent:direction:fromStore:isRoomInitialSync:] + 1022
	15  SwiftMatrixSDK                      0x0000000100e4d594 -[MXEventTimeline paginate:direction:onlyFromStore:complete:failure:] + 1125
	16  SwiftMatrixSDK                      0x0000000100f1ace2 _T0So15MXEventTimelineC14SwiftMatrixSDKE8paginateSo15MXHTTPOperationCSgSu_AC19MXTimelineDirectionO9directionSb13onlyFromStoreyAC10MXResponseOyytGc10completiontF + 386
	17  Seaglass                            0x0000000100056896 _T08Seaglass22MainViewRoomControllerC06scrollC9DidScrollySo14NSNotificationCF + 4438
	18  Seaglass                            0x0000000100056e0c _T08Seaglass22MainViewRoomControllerC06scrollC9DidScrollySo14NSNotificationCFTo + 60
	19  CoreFoundation                      0x00007fff3889fedc __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
	20  CoreFoundation                      0x00007fff3889fdaa _CFXRegistrationPost + 458
	21  CoreFoundation                      0x00007fff3889fae1 ___CFXNotificationPost_block_invoke + 225
	22  CoreFoundation                      0x00007fff3885d880 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1664
	23  CoreFoundation                      0x00007fff3885c9b7 _CFXNotificationPost + 599
	24  Foundation                          0x00007fff3a96c8c7 -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
	25  AppKit                              0x00007fff35efe3d1 -[NSClipView _immediateScrollToPoint:] + 614
	26  AppKit                              0x00007fff35efe0a7 -[NSClipView scrollToPoint:] + 232
	27  AppKit                              0x00007fff36819139 -[NSTableRowData _keepTopRowStable:andDoWorkUntilDone:] + 675
	28  AppKit                              0x00007fff3681941d -[NSTableRowData _automaticRowHeightsUpdateVisibleRowViews] + 161
	29  AppKit                              0x00007fff35f27971 -[NSTableRowData updateVisibleRowViews] + 261
	30  AppKit                              0x00007fff35f3c469 -[NSTableView _scrollViewDidChangeBounds:] + 38
	31  CoreFoundation                      0x00007fff3889fedc __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
	32  CoreFoundation                      0x00007fff3889fdaa _CFXRegistrationPost + 458
	33  CoreFoundation                      0x00007fff3889fae1 ___CFXNotificationPost_block_invoke + 225
	34  CoreFoundation                      0x00007fff3885d880 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1664
	35  CoreFoundation                      0x00007fff3885c9b7 _CFXNotificationPost + 599
	36  Foundation                          0x00007fff3a96c8c7 -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
	37  AppKit                              0x00007fff35efe8b9 -[NSView _postBoundsChangeNotification] + 103
	38  AppKit                              0x00007fff35efe7e8 -[NSView translateOriginToPoint:] + 261
	39  AppKit                              0x00007fff35efe383 -[NSClipView _immediateScrollToPoint:] + 536
	40  AppKit                              0x00007fff35efe0a7 -[NSClipView scrollToPoint:] + 232
	41  AppKit                              0x00007fff35f39b7f -[NSScrollView scrollClipView:toPoint:] + 75
	42  AppKit                              0x00007fff35ee4929 -[NSClipView _scrollTo:animateScroll:flashScrollerKnobs:] + 1273
	43  AppKit                              0x00007fff364e4750 -[_NSScrollingConcurrentMainThreadSynchronizer _scrollToCanonicalOrigin] + 935
	44  AppKit                              0x00007fff36080d74 -[_NSScrollingConcurrentMainThreadSynchronizer _synchronize:completionHandler:] + 174
	45  AppKit                              0x00007fff36080c94 __80-[_NSScrollingConcurrentMainThreadSynchronizer initWithSharedData:constantData:]_block_invoke + 145
	46  libdispatch.dylib                   0x0000000101ca2d8f _dispatch_client_callout + 8
	47  libdispatch.dylib                   0x0000000101cb73d2 _dispatch_continuation_pop + 940
	48  libdispatch.dylib                   0x0000000101ca548f _dispatch_source_invoke + 658
	49  libdispatch.dylib                   0x0000000101caf053 _dispatch_main_queue_callback_4CF + 1135
	50  CoreFoundation                      0x00007fff388c8c19 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	51  CoreFoundation                      0x00007fff3888adfa __CFRunLoopRun + 2586
	52  CoreFoundation                      0x00007fff3888a153 CFRunLoopRunSpecific + 483
	53  HIToolbox                           0x00007fff37b74d96 RunCurrentEventLoopInMode + 286
	54  HIToolbox                           0x00007fff37b74b06 ReceiveNextEventCommon + 613
	55  HIToolbox                           0x00007fff37b74884 _BlockUntilNextEventMatchingListInModeWithFilter + 64
	56  AppKit                              0x00007fff35e25a73 _DPSNextEvent + 2085
	57  AppKit                              0x00007fff365bbe34 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 3044
	58  AppKit                              0x00007fff35e1a885 -[NSApplication run] + 764
	59  AppKit                              0x00007fff35de9a72 NSApplicationMain + 804
	60  Seaglass                            0x0000000100076ded main + 13
	61  libdyld.dylib                       0x00007fff606cb015 start + 1
	62  ???                                 0x0000000000000003 0x0 + 3
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by ara4n** [comment]
this might be related: it happened on the third message i tried to send in #seaglass after opening the app.

```
2018-09-14 18:44:07.126983+0100 Seaglass[41589:3112988] [MXSession] Received 1 joined rooms, 0 invited rooms, 0 left rooms, 0 toDevice events in 280ms
2018-09-14 18:44:07.127177+0100 Seaglass[41589:3112988] *** Assertion failure in -[NSTableRowData removeRowsAtIndexes:withRowAnimation:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1668/TableView.subproj/NSTableRowData.m:7444
2018-09-14 18:44:07.150415+0100 Seaglass[41589:3112988] [General] An uncaught exception was raised
2018-09-14 18:44:07.150945+0100 Seaglass[41589:3112988] [General] removeRowsAtIndexes:withRowAnimation: can not happen while updating visible rows!
2018-09-14 18:44:07.151098+0100 Seaglass[41589:3112988] [General] (
	0   CoreFoundation                      0x00007fff3683b43d __exceptionPreprocess + 256
	1   libobjc.A.dylib                     0x00007fff62685720 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff3685608e +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x00007fff38bd155d -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 194
	4   AppKit                              0x00007fff33f48f04 -[NSTableRowData removeRowsAtIndexes:withRowAnimation:] + 350
	5   AppKit                              0x00007fff33f2c52a -[NSTableView _doUpdatedWorkWithHandler:] + 99
	6   AppKit                              0x00007fff33f48c59 -[NSTableView removeRowsAtIndexes:withAnimation:] + 139
	7   Seaglass                            0x00000001000f6ff3 _T08Seaglass15MatrixRoomCacheC7replaceySo7MXEventC_Si2attF + 1795
	8   Seaglass                            0x00000001000b7c84 _T08Seaglass14MatrixServicesC15subscribeToRoomySS6roomId_tFySo7MXEventC_05SwiftB3SDK19MXTimelineDirectionOSo11MXRoomStateCtcfU2_ + 17348
	9   Seaglass                            0x00000001000b81d5 _T08Seaglass14MatrixServicesC15subscribeToRoomySS6roomId_tFySo7MXEventC_05SwiftB3SDK19MXTimelineDirectionOSo11MXRoomStateCtcfU2_TA + 53
	10  SwiftMatrixSDK                      0x00000001015ba5db _T0So15MXEventTimelineC14SwiftMatrixSDKE14listenToEventsypSayAC0A4TypeOGSg_ySo0A0C_AC19MXTimelineDirectionOSo11MXRoomStateCtctFyAJSg_SC02__jK0VANSgtcfU_ + 251
	11  SwiftMatrixSDK                      0x00000001015ba671 _T0So15MXEventTimelineC14SwiftMatrixSDKE14listenToEventsypSayAC0A4TypeOGSg_ySo0A0C_AC19MXTimelineDirectionOSo11MXRoomStateCtctFyAJSg_SC02__jK0VANSgtcfU_TA + 17
	12  SwiftMatrixSDK                      0x00000001015ba81f _T0So7MXEventCSgSC21__MXTimelineDirectionVSo11MXRoomStateCSgIegxyx_AceHIeyByyy_TR + 95
	13  SwiftMatrixSDK                      0x000000010145c4e0 -[MXEventListener notify:direction:andCustomObject:] + 288
	14  SwiftMatrixSDK                      0x00000001014635ec -[MXEventTimeline notifyListeners:direction:] + 700
	15  SwiftMatrixSDK                      0x0000000101461c96 -[MXEventTimeline addEvent:direction:fromStore:isRoomInitialSync:] + 1782
	16  SwiftMatrixSDK                      0x00000001014606d1 -[MXEventTimeline handleJoinedRoomSync:] + 3009
	17  SwiftMatrixSDK                      0x00000001015586dc -[MXRoom handleJoinedRoomSync:] + 108
	18  SwiftMatrixSDK                      0x000000010158e553 __83-[MXSession serverSyncWithServerTimeout:success:failure:clientTimeout:setPresence:]_block_invoke + 2691
	19  SwiftMatrixSDK                      0x0000000101537014 __93-[MXRestClient syncFromToken:serverTimeout:clientTimeout:setPresence:filter:success:failure:]_block_invoke_3 + 36
	20  libdispatch.dylib                   0x000000010235d71b _dispatch_call_block_and_release + 12
	21  libdispatch.dylib                   0x000000010235e7f3 _dispatch_client_callout + 8
	22  libdispatch.dylib                   0x000000010236d3c0 _dispatch_main_queue_callback_4CF + 1909
	23  CoreFoundation                      0x00007fff36782147 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	24  CoreFoundation                      0x00007fff36781856 __CFRunLoopRun + 2335
	25  CoreFoundation                      0x00007fff36780ce4 CFRunLoopRunSpecific + 463
	26  UIFoundation                        0x00007fff5e9bbb85 -[NSHTMLReader _loadUsingWebKit] + 1916
	27  UIFoundation                        0x00007fff5e9beb8b -[NSHTMLReader attributedString] + 22
	28  UIFoundation                        0x00007fff5e9439b8 _NSReadAttributedStringFromURLOrData + 12698
	29  UIFoundation                        0x00007fff5e94078a -[NSAttributedString(NSAttributedStringUIFoundationAdditions) initWithData:options:documentAttributes:error:] + 117
	30  Seaglass                            0x000000010012b572 _T0So25NSMutableAttributedStringCAB10Foundation4DataV4data_s10DictionaryVySo012NSAttributedC0C6AppKitE24DocumentReadingOptionKeyVypG7optionss33AutoreleasingUnsafeMutablePointerVySo12NSDictionaryCSgGSg18documentAttributestKcfcTO + 194
	31  Seaglass                            0x000000010012a209 _T0So25NSMutableAttributedStringCAB10Foundation4DataV4data_s10DictionaryVySo012NSAttributedC0C6AppKitE24DocumentReadingOptionKeyVypG7optionss33AutoreleasingUnsafeMutablePointerVySo12NSDictionaryCSgGSg18documentAttributestKcfC + 89
	32  Seaglass                            0x00000001001291f7 _T0SS8SeaglassE26toAttributedStringFromHTMLSo012NSAttributedD0CSC15NSTextAlignmentO7justify_tF + 855
	33  Seaglass                            0x0000000100150e3b _T08Seaglass11RoomMessageC11textContentSSSg6string_So18NSAttributedStringCSg010attributedH0tyF + 3675
	34  Seaglass                            0x000000010002112a _T08Seaglass19RoomMessageIncomingC12viewWillDrawyyF + 26922
	35  Seaglass                            0x0000000100025a84 _T08Seaglass19RoomMessageIncomingC12viewWillDrawyyFTo + 36
	36  AppKit                              0x00007fff33e8717a -[NSTableCellView layout] + 92
	37  AppKit                              0x00007fff33dce6d9 _NSViewLayout + 587
	38  AppKit                              0x00007fff33dce080 -[NSView _layoutSubtreeWithOldSize:] + 446
	39  AppKit                              0x00007fff33dce2f9 -[NSView _layoutSubtreeWithOldSize:] + 1079
	40  AppKit                              0x00007fff33dcbb1a -[NSView _layoutSubtreeIfNeededAndAllowTemporaryEngine:] + 1358
	41  AppKit                              0x00007fff33e377a5 -[NSView _setDefaultKeyViewLoop] + 292
	42  AppKit                              0x00007fff33e9ecba -[NSTableRowData _updateKeyViewLoopforRowView:] + 55
	43  AppKit                              0x00007fff33e96391 -[NSTableRowData _initializeRowView:atRow:] + 397
	44  AppKit                              0x00007fff33e9566e -[NSTableRowData _preparedRowViewForRow:storageHandler:] + 154
	45  AppKit                              0x00007fff33e954ce -[NSTableRowData _addRowViewForVisibleRow:withPriorView:] + 296
	46  AppKit                              0x00007fff33e95300 -[NSTableRowData _addRowViewForVisibleRow:withPriorRowIndex:inDictionary:withRowAnimation:] + 404
	47  AppKit                              0x00007fff33e85626 -[NSTableRowData _updateVisibleRowEntries] + 1427
	48  AppKit                              0x00007fff33f2b702 __59-[NSTableRowData _automaticRowHeightsUpdateVisibleRowViews]_block_invoke.1498 + 97
	49  AppKit                              0x00007fff33f2b491 -[NSTableRowData _keepTopRowStable:andDoWorkUntilDone:] + 535
	50  AppKit                              0x00007fff33f2b268 -[NSTableRowData _automaticRowHeightsUpdateVisibleRowViews] + 143
	51  AppKit                              0x00007fff33e84fcc -[NSTableRowData updateVisibleRowViews] + 261
	52  AppKit                              0x00007fff33ecfc86 -[NSTableRowData prepareContentInRect:] + 88
	53  AppKit                              0x00007fff33ecfc1c -[NSTableView prepareContentInRect:] + 262
	54  AppKit                              0x00007fff33edd76f __38-[NSView _pullInExtraTilesForOverdraw]_block_invoke + 325
	55  AppKit                              0x00007fff33edcd70 -[NSView _performWorkOnTilesFromRect:renderedContentRect:maximumRect:scrollVelocity:handler:] + 1315
	56  AppKit                              0x00007fff33edc70b -[NSView _pullInExtraTilesForOverdraw] + 872
	57  AppKit                              0x00007fff33edc374 -[NSView _doIdlePrefetch] + 37
	58  CoreFoundation                      0x00007fff3679e405 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	59  CoreFoundation                      0x00007fff3679e3ab __CFRunLoopDoSource0 + 108
	60  CoreFoundation                      0x00007fff36781e51 __CFRunLoopDoSources0 + 195
	61  CoreFoundation                      0x00007fff367813fa __CFRunLoopRun + 1219
	62  CoreFoundation                      0x00007fff36780ce4 CFRunLoopRunSpecific + 463
	63  HIToolbox                           0x00007fff35a1a895 RunCurrentEventLoopInMode + 293
	64  HIToolbox                           0x00007fff35a1a5cb ReceiveNextEventCommon + 618
	65  HIToolbox                           0x00007fff35a1a348 _BlockUntilNextEventMatchingListInModeWithFilter + 64
	66  AppKit                              0x00007fff33cd78df _DPSNextEvent + 997
	67  AppKit                              0x00007fff33cd667e -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1362
	68  AppKit                              0x00007fff33cd06e1 -[NSApplication run] + 699
	69  AppKit                              0x00007fff33cbfe1b NSApplicationMain + 780
	70  Seaglass                            0x000000010007822d main + 13
	71  libdyld.dylib                       0x00007fff637520a1 start + 1
```

**Comment by neilalexander** [comment]
Seems to be fixed with e2bd2173402410f5d4151c2e44e948a3c3dbfec4. Will reopen if it happens again.
